bin/resty

```diff
@@ -150,20 +154,35 @@ mkdir $logs_dir or die "failed to mkdir $logs_dir: $!";
 my $conf_dir = File::Spec->catfile($prefix_dir, "conf");
 mkdir $conf_dir or die "failed to mkdir $conf_dir: $!";
 
-my $chunk_name;
+my $inline_lua = '';
+my $quoted_luafile;
 if (defined $src) {
-    $luafile = File::Spec->catfile($conf_dir, "a.lua");
-    open my $out, ">$luafile"
-        or die "Cannot open $luafile for writing: $!\n";
+    my $luainlinefile = File::Spec->catfile($conf_dir, "a.lua");
+    open my $out, ">$luainlinefile"
+        or die "Cannot open $luainlinefile for writing: $!\n";
     print $out $src;
     close $out;
-    $chunk_name = "=(command line -e)";
+    my $chunk_name = "=(command line -e)";
+    $quoted_luafile = quote_as_lua_str($luainlinefile);
 
-} else {
-    $chunk_name = "\@$luafile";
+    $inline_lua = <<"_EOC_";
+                local fname = $quoted_luafile
+                local f = assert(io.open(fname, "r"))
+                local chunk = f:read("*a")
+                inlinegen = assert(loadstring(chunk, "$chunk_name"))
```

Comment:
Let's avoid using Lua global variables here. Always use `local` to declare Lua variables unless absolutely necessary.